### ISSUE TITLE
Adding a third entry in Flow control presents an error 

### ISSUE THREAD

**Issue by yanushg** [curiefense/curiefense#379]
**Describe the bug**
When we add the third different entry under count by in flow control we get an error "Count-by entries must be unique" even if the are unique 
**To Reproduce**
1. Go to flow control 
2. Add another entry under count by "Attribute:country" 
3. Add another entry under count by "Attribute:Method" 

**Expected behavior**
We have chosen different entries so we should not get an error  
**Screenshots**
screenshot added 

**Desktop (please complete the following information):**
 - Browser chrome
 - Version 1.3.6

**Some more data** 
- we can save even though we have an error - the save btn enters loading state and stays in loading until refresh 

![Screen Shot 2021-06-20 at 14 07 09](https://user-images.githubusercontent.com/48092909/122672212-0c52f500-d1d3-11eb-91c8-c62855d81b3a.png)

**Comment by jdorfman** [comment]
Hi @yanushg! Thanks for creating this issue. We will look into it and get back to you ASAP.

**Comment by denis-reblaze** [comment]
Duplicate of #383

**Comment by jdorfman** [comment]
Once this is merged, we will close this issue out

**Comment by Aviv-Galmidi** [comment]
Closed in https://github.com/curiefense/curiefense/pull/420 (new PR as the first one was from a problematic branch)
